### ISSUE TITLE
Migrate from Popper (v2) to Floating UI (Popper successor)

### ISSUE THREAD

**Issue by strarsis** [atomiks/tippyjs#1071]
## Problem
Popper (v2) is now Floating UI (successor).

## Solution
It would be great if tippyjs can migrate from Popper (v2) to Floating UI.

**Comment by aeimi** [comment]
Will v7 be powered by Floating UI?

**Comment by atomiks** [comment]
I haven't had time to work on Tippy anymore. If you want to make a PR that switches to Floating UI, I can publish it as v7.

**Comment by aeimi** [comment]
I don't have the skill and time to do so. But what would you recommend. Still using tippy or just try to use some custom code to try to use floating ui? I'm talking here about vanilla js. Is it worth it to drop tippy for new shiny thing? Again, just Vanilla JS.

**Comment by atomiks** [comment]
@amdiv I think it depends what you're doing with it. If you're just using it for tooltips and simple popovers, definitely not. If you're doing more advanced positioning and need some of the new features Floating UI offers, then maybe. (It also has some bug fixes, but if you aren't noticing them then it doesn't matter). 

If Tippy is working fine for you I don't see a reason to change right now

**Comment by aeimi** [comment]
Cool. Tippy is such an amazing piece of work. It would be sad for me to see it just as 'outdated'. Pushed out by shiny new thing. Now I get why Floating UI website have the same style as Tippy's. You're the man behind it all. Thank you.

**Comment by sunwrobert** [comment]
@atomiks Are there noticeable performance improvements between using Popper (TippyJS v4) <-> PopperV2 (TippyJS v6) <-> Floating UI?

**Comment by vlrevolution** [comment]
I'd love to see this implemented as well. The new floating UI has some neat tricks up its sleeves

There's a migration guide for those who have time to work on it:
https://floating-ui.com/docs/migration

**Comment by mulder999** [comment]
It is sad to see tippyJS is no longer supported/updated. What alternatives are you using ?

**Comment by vlrevolution** [comment]
> It is sad to see tippyJS is no longer supported/updated. What alternatives are you using ?

I mean tippyjs is stable .. I'm still using it and added my own implementation to have dynamic scroll when content is too big

**Comment by mulder999** [comment]
We also rely on it; however, since it’s no longer maintained, we need to take action—either by supporting it ourselves or planning for a replacement.

**Comment by vlrevolution** [comment]
Yeah, you are right. The way to go would be to rewrite to use floating ui
